NightscoutServiceKitTests/OTPManagerTestCase.swift

```diff
@@ -13,99 +13,131 @@ import XCTest
 class OTPManagerTestCase: XCTestCase {
```

Comment:
Some test updates and also I refactored the test data modeling to be more flexible as these tests grow.